tensorflow/core/lib/jpeg/jpeg_mem.cc

```diff
@@ -540,7 +540,7 @@ bool CompressInternal(const uint8* srcdata, int width, int height,
         row_pointer[0] = reinterpret_cast<JSAMPLE*>(const_cast<JSAMPLE*>(r));
       }
     }
-    CHECK_EQ(jpeg_write_scanlines(&cinfo, row_pointer, 1), 1);
+    CHECK_EQ(jpeg_write_scanlines(&cinfo, row_pointer, 1), 1u);
```

Comment:
FYI, please note that `1u` is correct since `JDIMENSION` is `unsigned int`, not `size_t`.
```
typedef unsigned int JDIMENSION;
```